bot-components/GitHub_subscriptions.ml

```diff
@@ -7,17 +7,22 @@ type issue = {owner: string; repo: string; number: int}
 
 type issue_info =
   { issue: issue
+  ; title: string
   ; id: string
   ; user: string
   ; labels: string list
   ; milestoned: bool
   ; pull_request: bool
-  ; body: string option }
+  ; body: string option
+  ; milestone_id: string option
```

Comment:
I don't think that we need to know the precise milestone for this use case, so maybe the `milestoned` field was already enough.